application/src/main/java/org/togetherjava/tjbot/commands/system/BotCore.java

```diff
@@ -290,6 +308,40 @@ public void onUserContextInteraction(final UserContextInteractionEvent event) {
                 UserContextCommand.class).onUserContext(event));
     }
 
+    /**
+     * Returns the {@link ComponentId} of the given componentId. If the component has expired, the
+     * user gets a message stating that.
+     *
+     * @param event the {@link IReplyCallback event} to reply to
+     * @param componentId the component's ID
+     *
+     * @return the relating {@link ComponentId}
+     */
+    @Nullable
+    public ComponentId handleGetComponentId(IReplyCallback event, String componentId) {
```

Comment:
i think the method would be easier to understand if it returns `Optional`. (it seems u already made a bug bc its nullable and u didnt notice - see other comment).